### PR TITLE
Toggle header items when hitting the same hotkey repeatedly

### PR DESCRIPTION
**Pull request by PabloSichert**
#1808

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -280,32 +288,39 @@ class Header extends Component {
...
+      isUDOpen: false,
+      isSideListShow: false,
+      tooltipOpen: '',
+    };
+
+    if (clickedItem) {
+      delete state[clickedItem];
+    }
+
+    state.isSubheaderShow =
```
Comment (siemiatj) [inline src/components/header/Header.js]: isn't omitting `setState` considered a bad practice in the react world ?

File: src/components/keyshortcuts/GlobalContextShortcuts.js [hunk 2]
```diff
@@ -1,16 +1,51 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```
Comment (siemiatj) [inline src/components/keyshortcuts/GlobalContextShortcuts.js]: 👍

File: src/components/header/Header.js [hunk 3]
```diff
@@ -103,10 +103,18 @@ class Header extends Component {
...
 
+  handleInboxToggle = () => {
+    this.setState({ isInboxOpen: !this.state.isInboxOpen });
+  };
+
   handleUDOpen = state => {
     this.setState({ isUDOpen: !!state });
   };
 
+  handleUDToggle = () => {
```
Comment (siemiatj) [inline src/components/header/Header.js]: Can you add a comment what UD stands for ? I assume it's User's Dropdown ?

File: src/components/header/Header.js (hunk 1 above)
Comment (siemiatj) [reply src/components/header/Header.js]: my bad, carry on ;)

File: src/components/header/Header.js (hunk 3 above)
Comment (PabloSichert) [reply src/components/header/Header.js]: I have no idea, just went with the existing naming in the code. That illustrates your point well, though 😄

**Pull request merged**